en/guide/dronecore_extensions.md

```diff
@@ -91,5 +94,6 @@ sudo make default install   # sudo needed to install files into system directori
 
 ### Locking/Unlocking the SDK
 
-Functionality to deliver in: https://github.com/dronecore/DroneCore/pull/139
+Functionality delivered in: https://github.com/dronecore/DroneCore/pull/139

```

Comment:
I am assuming that this API still doesn't need to be documented!

Reply:
Right.